text/3463-crates-io-policy-update.md

```diff
@@ -0,0 +1,233 @@
+  minors (see <https://docs.github.com/en/site-policy/acceptable-use-policies/github-sexually-obscene-content>)
+- is false, inaccurate, or intentionally deceptive information and likely to
+  adversely affect the public interest (including health, safety, election
+  integrity, and civic participation)
+- is off-topic, or interacts with platform features in a way that significantly
+  or repeatedly disrupts the experience of other users
+- exists only to reserve a name for a prolonged period of time (often called 
+  "name squatting") without having any genuine functionality, purpose, or
+  significant development activity on the corresponding repository
+- is related to buying, selling, or otherwise trading of package names or any
+  other names on crates.io for money or other compensation
+- impersonates any person or entity, including through false association with
+  crates.io, or by fraudulently misrepresenting your identity or site's purpose
+- is related to inauthentic interactions, such as fake accounts and automated
+  inauthentic activity
+- is using our servers for any form of excessive automated bulk activity, to
+  place undue burden on our servers through automated means, or to relay any
+  form of unsolicited advertising or solicitation through our servers, such as
+  get-rich-quick schemes
+- is other automated excessive bulk activity and coordinated inauthentic
+  activity, such as
+    - spamming
+    - cryptocurrency mining
+- is not functionally compatible with the cargo build tool (for example, a
+  "package" cannot simply be a PNG or JPEG image, a movie file, or a text
+  document uploaded directly to the registry)
+- is abusing the package index for purposes it was not intended
+
+You are responsible for using crates.io in compliance with all applicable laws,
+regulations, and all of our policies. These policies may be updated from time to
+time. We will interpret our policies and resolve disputes in favor of protecting
+users as a whole. The crates.io team reserves the possibility to evaluate each
+instance on a case-by-case basis.
+
+For issues such as DMCA violations, or trademark and copyright infringements,
+the crates.io team will respect the legal decisions of the
+[Rust Foundation](https://rustfoundation.org/) as the official legal entity
+providing the crates.io service.
+
+
+## Package Ownership
+
+crates.io has a first-come, first-serve policy on crate names. Upon publishing a
+package, the publisher will be made owner of the package on crates.io.
+
+If you want to take over a package, we require you to first try and contact the
+current owner directly. If the current owner agrees, they can add you as an
+owner of the crate, and you can then remove them, if necessary. If the current
+owner is not reachable or has not published any contact information the
+crates.io team may reach out to help mediate the process of the ownership
+transfer.
```

Comment:
I'm a huge fan of anti-squatting mechanisms, but this seems like it goes a step further than that. If a crate *does* exist and *does* have a legitimate purpose (and is not just squatted), I don't think it's a good idea in general for crates.io to be making a determination about whether the maintainer is sufficiently active to be "allowed" to keep it, nor should it be making a determination about suitable new maintainers.